.pre-commit-config.yaml

```
@@ -32,3 +32,9 @@ repos:
         - id: rst-backticks
         - id: rst-directive-colons
         - id: rst-inline-touching-normal
+
+  - repo: https://github.com/kynan/nbstripout
+    rev: 0.7.1
+    hooks:
+      - id: nbstripout
+        args: ['--keep-output']
```

I think it's worth removing empty cells as well:
```suggestion
        args: ['--keep-output', '--drop-empty-cells']
```